js/src/components/query-builder/query-builder.component.vue

```diff
@@ -33,8 +30,9 @@
         query: this.data.query ? this.data.query : {},
         rules: this.data.rules ? this.data.rules : [],
         name: this.data.name ? this.data.name : '',
-        maxDepth: this.data.maxDepth ? ((this.data.maxDepth <= 5) ? this.data.maxDepth : 5 ): 1,
+        maxDepth: this.data.maxDepth ? ((this.data.maxDepth <= 10) ? this.data.maxDepth : 10 ): 1,
```

Comment:
why limit on two places?

Reply:
Just a safety on the component side.

Reply:
So with 100 the frontend will catch on fire? :D

Reply:
I think this safety check should be removed (it checks if data from php are ok?) and fallback to 10 if not specified. Simple.

Reply:
because the query can come from anywhere. If stored in DB for example, we may not have control over it. I prefer to limit it here and 10 levels should be quite enough.